pkg/registry/core/seed/strategy_test.go

```diff
@@ -50,5 +50,56 @@ var _ = Describe("Strategy", func() {
 			strategy.PrepareForUpdate(ctx, newSeed, oldSeed)
 			Expect(newSeed.Generation).To(Equal(oldSeed.Generation + 1))
 		})
+
+		Context("syncDependencyWatchdogSettings", func() {
+			BeforeEach(func() {
+				oldSeed.Spec = core.SeedSpec{
+					Settings: &core.SeedSettings{
+						DependencyWatchdog: &core.SeedSettingDependencyWatchdog{},
+					},
+				}
+				newSeed.Spec = core.SeedSpec{
+					Settings: &core.SeedSettings{
+						DependencyWatchdog: &core.SeedSettingDependencyWatchdog{},
+					},
+				}
+			})
+			It("should default new and deprecated fields for prober and weeder when both aren't specified", func() {
```

Comment:
Readability: Please leave a newline between the blocks.

```suggestion

			It("should default new and deprecated fields for prober and weeder when both aren't specified", func() {
```

Please repeat the same below for the further `It`s